### ISSUE TITLE
need version compatibility for windows 2022

### ISSUE THREAD

**Issue by thdrnsdk** [kubernetes-csi/livenessprobe#243]
in the repo dockerfile.windows
there is base, addon image is windwos images "1809"

Images created in this way do not seem to work on Windows 2022 servers due to version compatibility issues.

**Comment by mauriciopoppe** [comment]
Fortunately the values in https://github.com/kubernetes-csi/livenessprobe/blob/master/Dockerfile.Windows#L16-L17 can be overridden so you're looking at the default values, when we build the images we actually build them with different values seen in https://github.com/kubernetes-csi/livenessprobe/blob/ebd49b57031ab90f6f376f1472884948f381558c/release-tools/prow.sh#L81, the resulting image is a multi-arch image (a docker manifest) that the node will use to determine the right OS/arch combination to download based on the manifest.

In short, I believe we produce image compatible with Windows 2022 too.
